docs/components/bottom_navigation.md

```diff
@@ -0,0 +1,293 @@
+
+The following example shows a bottom navigation bar with four icons:
+
+* favorites
+* music note
+* places
+* news
+
+<img src="assets/bottom_navigation/flutter-bottom-nav-regular.png" alt="bottom navigation bar with favorites, music note, places, and news icons">
+
+```dart
+BottomNavigationBar(
+  type: BottomNavigationBarType.fixed,
+  backgroundColor: Color(0xFF6200EE),
+  selectedItemColor: Colors.white,
+  unselectedItemColor: Colors.white.withOpacity(.60),
+  selectedFontSize: 14,
+  unselectedFontSize: 14,
+  onTap: (value) {
+    // Respond to item press.
+  },
+  items: [
+    BottomNavigationBarItem(
+      title: Text('Favorites'),
+      icon: Icon(Icons.favorite),
+    ),
+    BottomNavigationBarItem(
+      title: Text('Music'),
+      icon: Icon(Icons.music_note),
+    ),
+    BottomNavigationBarItem(
+      title: Text('Places'),
+      icon: Icon(Icons.location_on),
+    ),
+    BottomNavigationBarItem(
+      title: Text('News'),
+      icon: Icon(Icons.library_books),
+    ),
+  ],
+)
+```
+
+### Anatomy and key properties
+
+The following is an anatomy diagram for the bottom navigation bar:
+
+![bottom nav anatomy diagram](assets/bottom_navigation/bottom-nav-anatomy.png)
+
+1. Container
+1. Inactive icon
+1. Inactive text label
+1. Active icon
+1. Active text label
+
+**Container attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Color** | `backgroundColor` |
+| **Stroke color** | N/A |
+| **Stroke width** | N/A |
+| **Shape** | N/A |
+| **Elevation** | `elevation` |
+| **Ripple color** | N/A |
+| **Disabled color** | N/A |
+| **Disabled elevation** | N/A |
+
+
+**Inactive icon attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Icon** | `icon` |
+| **Color** | `unselectedItemColor` |
+| **Size** | `size` on `unselectedIconTheme` |
+
+
+
+**Inactive text label attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Text label** | `title` |
+| **Color** | `unselectedItemColor` |
+| **Disabled Text Color** | N/A |
+| **Typography** | `unselectedLabelStyle` |
+
+**Active icon attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Icon** | `icon` |
+| **Color** | `selectedItemColor` |
+| **Size** | `size` on `selectedIconTheme` |
+
+
+
+**Active text label attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Text label** | `title`|
+| **Color** | `selectedItemColor` |
+| **Disabled Text Color** | N/A |
+| **Typography** | `selectedLabelStyle` |
+
+## Theming a bottom nav bar
+
+The following example shows a bottom navigation bar with the [Material Shrine Theme](https://material.io/design/material-studies/shrine.html).
+
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/bottom_navigation/theme)
+
+![FABs with Shrine theming](assets/bottom_navigation/flutter-bottom-nav-themed.png)
```

Comment:
Is this the correct link title?

Reply:
Guessing this should be `Bottom Navigation bar with Shrine theming`

Reply:
Yes link is right but text is not, updated.